samples/start-app/python-fastapi/app/usecase/openid/callback.py

```diff
@@ -0,0 +1,72 @@
+from app.persistence import userRepository
+from app.model import schemas
+from app.dependencies import get_config, build_user_agent
+from app.usecase.callback import callback_usecase
+from app.utils.codec import encoder, decoder
+from urllib.parse import urljoin
+import requests
+import jwt
+from jwt import decode, get_unverified_header
+from jwt.algorithms import get_default_algorithms
+
+
+openid_public_key = '/connect/apps/v1/openid/public-key';
+
+def callback_usecase_with_openid(request, db, session):
+    response = callback_usecase(request, db, session)
+
+    pim_url = session.headers['pim_url']
+    if pim_url == '':
+        raise ValueError('No PIM url in session')
```

Comment:
Maybe you could try to use the one in environment variables ?

Reply:
this function does not exist anymore